src/styles/commentCard.css

```diff
@@ -0,0 +1,35 @@
+.comment-card {
+    background-color: #5dd769;
+    color: #000;
+    border-radius: 8px;
+    padding: 16px;
+    margin-bottom: 16px;
+    text-align: center;
+    max-width: 400px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+  
```

Comment:
If you've set a default size for font, a standard p tag will take that sizing, you don't need to specify.